packages/components/src/internal/actions.ts

```diff
@@ -535,6 +535,32 @@ export function getExportParams(
     return params;
 }
 
+export async function exportTabsXlsx(filename:string, models: SchemaQuery[]): Promise<void>
```

Comment:
A few comments.

1. This method doesn't need to be marked as `async` as it it creating a new `Promise` and returning it explicitly.
2. For failures at the request level we tend to do a `console.error()`.
3. Consider using `Utils.getCallbackWrapper()` and  `resolveErrorMessage()` rather than parsing the error message yourself.

Here is my recommendation:

```tsx
export function exportTabsXlsx(filename: string, tabModels: SchemaQuery[]): Promise<void> {
    return new Promise((resolve, reject) => {
        Ajax.request({
            url: ActionURL.buildURL('query', 'exportTabsXlsx.api'),
            method: 'POST',
            jsonData: {
                filename,
                tabModels,
            },
            downloadFile: true,
            success: () => {
                resolve();
            },
            failure: Utils.getCallbackWrapper(error => {
                console.error('Failed to export tabular data', error);
                reject(resolveErrorMessage(error) ?? 'Unexpected error while exporting selected tabs.');
            }),
        });
    });
}
```